faq/play2-instances.mdx

```diff
@@ -0,0 +1,29 @@
+---
+meta:
+  title: PLAY2 Instances
+  description: Discover Scaleway PLAY2 Instances.
+content:
+  h1: PLAY2 Instances
+  paragraph: Discover Scaleway PLAY2 Instances.
+image: play2-instances.webp
+---
+
+## What are PLAY2 Instances?
+Scaleway’s latest generation PLAY2 Instances are designed for testing and development purposes.
+PLAY2 Instances are the second generation of Development Instances after DEV1. PLAY2 Instances are small and agile with adequate performance
```

Comment:
I don't think we should say "adequate", it doesn't sound right. Maybe "good"? Or else "...are small and agile and meet performance requirements for standard testing and development scenarios". ?